integration_tests/run_pyspark_from_build.sh

```diff
@@ -175,7 +175,10 @@ else
         TEST_PARALLEL_OPTS=("-n" "$TEST_PARALLEL")
     fi
 
-    RUN_DIR=${RUN_DIR-$(mktemp -p "$SCRIPTPATH"/target -d run_dir-$(date +%Y%m%d%H%M%S)-XXXX)}
+    TARGET_DIR="$SCRIPTPATH"/target
```

Comment:
technically this dir does not need to be created if RUN_DIR is provided by the caller. But it's not a big deal and can be cleaned up in a follow up issue